server/src/controllers/game.controller.js

```diff
@@ -76,7 +76,7 @@ class GameController {
       const payload = {
         event: "join_game",
         permission,
-        player: opponent,
+        player: updated.data.opponent,
```

Comment:
switch this to ```opponent``` only, not ``updated.data.opponent``

Reply:
I didn't work on that tho
lemme pull dev upstream if it's from the new merge

Reply:
okay boss
fixed that